inc/advanced-navigation.php

```diff
@@ -130,7 +130,7 @@ function form($filter_arg = null) {
 			$filter_taxonomy = $filter_arg['taxonomy'];
 		endif;
 
-		if(isset($filter_arg['depth']) && !empty($filter_arg['depth'])):
+		if(isset($filter_arg['depth']) && !is_null($filter_arg['depth'])):
```

Comment:
Yes, I also thought that. However, during the test, the result didn't get as expected when I set depth = 0; Then, I added is_null(). 
Anyways, this code "&& !is_null($filter_arg['depth'])" can remove it because it doesn't need anymore since the top level is 1.